### ISSUE TITLE
Autogenerated script for Teleport agent installation to support armv7l 

### ISSUE THREAD

**Issue by r4ndomuser** [gravitational/teleport#8443]
### What
Add support for armv7l for the automatic script for teleport agent install

### How
Add support (build dedicated arch, modify script, test)

### Why
To run a node on Raspberry

### Workaround
Manually build from source and setup node

Currently (2021-10-01) automatic script doesn't support this.

```
cat /etc/os-release 
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
```

```
uname -m
armv7l

```

```
uname -a
Linux oh 5.10.17-v7l+ #1414 SMP Fri Apr 30 13:20:47 BST 2021 armv7l GNU/Linux
```

```
sudo bash -c "$(curl -fsSL https://***.teleport.sh/scripts/****/install-node.sh)"
2021-10-01 14:43:00 EEST [teleport-installer] TELEPORT_VERSION: 7.1.1
2021-10-01 14:43:00 EEST [teleport-installer] TARGET_HOSTNAME: ***.teleport.sh
2021-10-01 14:43:00 EEST [teleport-installer] TARGET_PORT: 443
2021-10-01 14:43:00 EEST [teleport-installer] JOIN_TOKEN: ***
2021-10-01 14:43:00 EEST [teleport-installer] CA_PIN_HASH: sha256:***
2021-10-01 14:43:00 EEST [teleport-installer] Checking TCP connectivity to Teleport server (***.teleport.sh:443)
2021-10-01 14:43:00 EEST [teleport-installer] Connectivity to Teleport server (via nc) looks good
2021-10-01 14:43:00 EEST [teleport-installer] Detected host: linux-gnueabihf, using Teleport binary type linux
2021-10-01 14:43:00 EEST [teleport-installer] Detected arch: armv7l, using Teleport arch arm
2021-10-01 14:43:00 EEST [teleport-installer] Detected distro type: debian
2021-10-01 14:43:00 EEST [teleport-installer] Using Teleport distribution: deb
2021-10-01 14:43:00 EEST [teleport-installer] Created temp dir /tmp/teleport-bOFpatWlO9
2021-10-01 14:43:01 EEST [teleport-installer] Downloading Teleport deb release 7.1.1
2021-10-01 14:43:01 EEST [teleport-installer] Running curl -fsSL --retry 5 --retry-delay 5 https://get.gravitational.com/teleport_7.1.1_arm.deb
2021-10-01 14:43:01 EEST [teleport-installer] Downloading to /tmp/teleport-bOFpatWlO9/teleport_7.1.1_arm.deb
2021-10-01 14:43:14 EEST [teleport-installer] Downloaded file size: 47786484 bytes
2021-10-01 14:43:14 EEST [teleport-installer] Will use shasum -a 256 to validate the checksum of the downloaded file
2021-10-01 14:43:25 EEST [teleport-installer] The downloaded file's checksum validated correctly
2021-10-01 14:43:25 EEST [teleport-installer] Using dpkg to install /tmp/teleport-bOFpatWlO9/teleport_7.1.1_arm.deb
dpkg: error processing archive /tmp/teleport-bOFpatWlO9/teleport_7.1.1_arm.deb (--install):
 package architecture (arm) does not match system (armhf)
Errors were encountered while processing:
 /tmp/teleport-bOFpatWlO9/teleport_7.1.1_arm.deb
```

**Comment by benarent** [comment]
In the meantime, @r4ndomuser does https://get.gravitational.com/teleport-v7.2.0-linux-arm-bin.tar.gz or https://get.gravitational.com/teleport_7.2.0_arm.deb work for you? ( so you don't need to build the image? )

**Comment by r4ndomuser** [comment]
Unfortunately, it's a different arch...
![image](https://user-images.githubusercontent.com/77339076/135803343-0bf80a58-f688-4130-810a-da8a2a3889c9.png)

**Comment by Valien** [comment]
Can confirm this breaks on Pi's. Tried to upgrade from `v6.0.1` on my pi and getting the same error.

**Comment by ToniCipriani** [comment]
Not working fwith the latest `v8.1.1` package as well:

```
 sudo dpkg -i ./teleport_8.1.1_arm.deb
dpkg: error processing archive ./teleport_8.1.1_arm.deb (--install):
 package architecture (arm) does not match system (armhf)
Errors were encountered while processing:
 ./teleport_8.1.1_arm.deb
```

**Comment by espadolini** [comment]
Fixed by #11318
